app/controllers/users_controller.rb

```diff
@@ -1,13 +1,17 @@
 class UsersController < ApplicationController
   before_action :set_user, only: [:show, :edit, :update]
 
+  load_and_authorize_resource
+
   def index
     @users = User.all
   end
 
   def show
     @cards = @user.latest_cards
     @liked = @user.latest_likes
+    @total_likes = @user.total_likes
+    @total_cards = @user.total_cards
```

Comment:
Why not just call these directly in the template? I guess we could stick to `@user` for them all.. Unless you prefer the semantic purity - as I do ;p